src/rpc/methods/eth.rs

```diff
@@ -72,9 +156,47 @@ impl Address {
         }
     }
 
+    pub fn from_filecoin_address(addr: &FilecoinAddress) -> Result<Self> {
```

Comment:
Is this method unit-tested?

Reply:
Could you link here the corresponding Go impl?

Reply:
Link added.